builder/bundles.go

```diff
@@ -323,11 +326,14 @@ func queryRpmName(packageCmd []string, pkgName string, repo string) string {
 	queryStringRpm = append(queryStringRpm, pkgName)
 	outBuf, _ := helpers.RunCommandOutputEnv(queryStringRpm[0], queryStringRpm[1:], []string{"LC_ALL=en_US.UTF-8"})
 	if outBuf.String() != "" {
-		rpm := strings.Split(outBuf.String(), "\n")
-		_, file := filepath.Split(rpm[0])
-		return file
+		out := strings.Split(outBuf.String(), "\n")
+		pURL, err := url.Parse(out[0])
+		if err == nil { // obtain the path without the url scheme
+			pURL.Scheme = ""
+			rpmFullPath = pURL.String()
+		}
 	}
-	return ""
```

Comment:
we should create an error string if outBuf.String() is empty. os.stat() won't catch the error as in this case as rpmFullPath will be empty string

Reply:
os.stat() of empty string gives file not found err.

Reply:
Is there a reason we don't return an error here instead of during os.stat() when the rpm cannot be resolved?

Reply:
It is mainly for fewer lines of code and the assumption that there would not be a performance impact as the chances of os.Stat("") during this stage is very less.

But I think since you both have brought up this concern, I will return an err if the rpm is not resolved.